Sources/OpenAPI/.create-api.yml

```diff
@@ -29,17 +29,18 @@ paths:
 
 entities:
   isGeneratingCustomCodingKeys: false
+  isGeneratingIdentifiableConformance: true
```

Comment:
Funny, I tried this locally myself but it didn't result in any changes. I wonder howcome it worked for you! Super happy with this though 🙌

Reply:
Ha strange.. It could be that you used a version prior to 0.0.5? 

Alternatively, it could have just been a typo/issue with the yaml? I ran into this issue a few times so the next version will warn you if the options aren't known (CreateAPI/CreateAPI#77)

Anyway, thanks for merging!